services/core-api/app/api/services/issue_to_orgbook_service.py

```diff
@@ -0,0 +1,102 @@
+                })
+            if response.status_code == 200:
+                self.issuer_controller_ready = True
+                current_app.logger.info('Issuer Controller ready and secret-key correct!')
+                return
+            elif response.status_code != 503:
+                current_app.logger.error('Livenesscheck failed: ' + str(response.status_code))
+            time.sleep(.500 * retry_count)
+        current_app.logger.error('Issuer Controller liveness check failed, nothing will be issued')
+
+    def publish_schema(self, new_schema, new_schema_version):
+        raise NotImplementedError(
+            """The MDT issuer controller publishes schema on init, to update your VC schema,
+                send them a message. This can be done dynamically in the future once MDT implements it"""
+        )
+
+    def inspect_current_schema(self):
+        raise NotImplementedError(
+            """Hopefully this exists, the MDT issuer controller should be able to inspect
+                and return the current schema it is expecting/allowing. 
+            
+                Inspecting old schemas can be done by looking at the hyperledger directly
+                production -> http://prod.bcovrin.vonx.io/browse/domain?page=1&query=&txn_type=101
+                test - http://test.bcovrin.vonx.io/browse/domain?page=1&query=&txn_type=101
+                dev  - http://dev.bcovrin.vonx.io/browse/domain?page=1&query=&txn_type=101
+                """)
+
+    def issue_permit_amendment_vc(self, permit_amendment):
+        if not self.issuer_controller_ready:
+            current_app.logger.warning(
+                'Issuer Controller liveness check failed on init, this call did nothing')
+        permittee_orgbook_entity = permit_amendment.permit.permittee_appointments[
+            0].party.party_orgbook_entity
+        if not permittee_orgbook_entity:
+            current_app.logger.warning(
+                'skipping issue_permit_amendment_vc, permittee not link to orgbook business')
+            return
+
+        payload = json.dumps([{
+            "schema": self.vc_schema_name,
+            "version": self.vc_schema_version,
+            "attributes": {
+                "permit_id": str(permit_amendment.permit_guid),
+                "registration_id": permittee_orgbook_entity.registration_id,
+                "permit_no": permit_amendment.permit.permit_no,
+                "mine_no": permit_amendment.mine.mine_no,
+                "mine_class": 'major' if permit_amendment.mine.major_mine_ind else 'regional',
+                "latitude": str(permit_amendment.mine.latitude),
+                "longitude": str(permit_amendment.mine.longitude),
+                "issued_date": str(permit_amendment.issue_date),
+                "effective_date": str(permit_amendment.issue_date),
+                "authorization_end_date": str(permit_amendment.authorization_end_date),
+                "inspector_name": "Best Inspector"
```

Comment:
This inspector name needs to be changed?

Reply:
Good eye.